tomviz/PipelineExecutor.cxx

```diff
@@ -176,6 +177,19 @@ DockerPipelineExecutor::DockerPipelineExecutor(Pipeline* pipeline)
 
 DockerPipelineExecutor::~DockerPipelineExecutor() = default;
 
+QString DockerPipelineExecutor::originalFileName()
+{
+  QString ext = ".emd";
+  auto* dataSource = pipeline()->dataSource();
+  if (dataSource->darkData() && dataSource->whiteData())
```

Comment:
@cjh1 Here is the function I'm using to determine whether docker writes out to emd or Data Exchange. Because it was simpler, I just checked for the presence of dark and white data to determine whether we should write to Data Exchange. However, if it's preferable, I can instead check the original file extension, and then open up the original file and make sure that "/exchange/data" exists. Let me know.

Reply:
No, that should work fine.